packages/rum/src/rum.ts

```diff
@@ -503,10 +504,14 @@ function trackPerformanceTiming(
 
 export function handleResourceEntry(
   configuration: Configuration,
-  entry: PerformanceResourceTiming,
+  lifeCycle: LifeCycle,
+  session: RumSession,
   handler: (startTime: number, event: RumResourceEvent) => void,
-  lifeCycle: LifeCycle
+  entry: PerformanceResourceTiming
 ) {
+  if (!session.isTrackedWithResource()) {
+    return
+  }
   if (!isValidResource(entry.name, configuration)) {
     return
   }
```

Comment:
why not a single if?

Reply:
This is the same condition as `trackRequests`, and I find having two separate conditions for two different concerns easier to read, but I could:

* join them in a single condition
* factorize it with `trackRequests` into a separate `shouldTrackResource` function

WDYT?

Reply:
+1 for `shouldTrackResource`